alvin/script.js

```diff
@@ -0,0 +1,241 @@
+
+
+// https://www.programiz.com/javascript/examples/get-random-item
+// get item from array
+
+function randomWord(words) {
+    const random = Math.floor(Math.random() * words.length);
+
+    const item = words[random];
+
+    return item;
+}
+
+const chosenWord = randomWord(words);
+console.log(chosenWord);
```

Comment:
```suggestion
//console.log(chosenWord);
```

Aim to have an empty browser console.